pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -270,34 +271,45 @@ def fetch_mirror(remote):
     return None
 
 
-def fetch_remote_url(remote):
+def fetch_remote_url(remote, custom_url=None):
     """Fetch a single remote from which can be content synced."""
 
-    def normalize_url(url):
-        return url.rstrip("/") + "/"
+    def normalize_url(url_to_normalize):
+        return url_to_normalize.rstrip("/") + "/"
+
+    url = custom_url or remote.url
 
     try:
-        normalized_remote_url = normalize_url(remote.url)
+        normalized_remote_url = normalize_url(url)
         get_repomd_file(remote, normalized_remote_url)
         # just check if the metadata exists
         return normalized_remote_url
     except ClientResponseError as exc:
-        log.info(
-            _("Attempting to resolve a true url from potential mirrolist url '{}'").format(
-                remote.url
+        # If URL is passed by parameter 'url' it is called from ACS refresh
+        # which doesn't support mirror lists.
+        if custom_url:
+            raise ValueError(
```

Comment:
At this point we do not really know whether the provided url is just bad url or a mirrorlist, we just know that if the custom_url was provided  and for some reason we have caught the ClientResponseError we can raise the value error right away and not try to call into `fetch_mirror`.

tldr;
```
if custom_url:
    raise ValueError(_("Remote URL {} for Alternate content source is invalid").format(custom_url))
```